cypress/e2e/read-receipts/missing-referents.spec.ts

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ * # High Level Read Receipt Tests
+ *
+ * Tips for writing these tests:
+ *
+ * * Break up your tests into the smallest test case possible. The purpose of
+ *   these tests is to understand hard-to-find bugs, so small tests are necessary.
+ *   We know that Cypress recommends combining tests together for performance, but
+ *   that will frustrate our goals here. (We will need to find a different way to
+ *   reduce CI time.)
+ *
+ * * Try to assert something after every action, to make sure it has completed.
+ *   E.g.:
+ *   markAsRead(room2);
+ *   assertRead(room2);
+ *   You should especially follow this rule if you are jumping to a different
+ *   room or similar straight afterwards.
+ *
+ * * Use assertStillRead() if you are asserting something is read when it was
+ *   also read before. This waits a little while to make sure you're not getting a
+ *   false positive.
+ */
```

Comment:
it seems unwise do have 7 copies of this text. They are basically guaranteed to drift out of sync.

Can we move it to a separate text file (`cypress/e2e/read-receipts/readme.md` ?) and just refer to it here?

Reply:
Good call, done in f4441da985